### DISCUSSION TITLE
Thanks for your script

### DISCUSSION THREAD

**Discussion by MilesTEG1** [007revad/Synology_HDD_db#80] [General]
Hello,
I wanted to thank you for your script, it helped me to restore my NVMe volume that as been killed by DSM 7.2 update (after I reboot my DS920+ after the update process finished).
My RAID1 of NVMe with Crucial P3 500GB (CT500P3SSD8) was working fine with 7.1.x, but the DSM 7.2 said that they aren't compatible, and all the work I've done to build a raid1 has vanished... (see my french version of tutoriel on my gist).

By writing this, I'm looking at my screenshots, and maybe your script wasn't mandatory to get my RAID1 working... but nevertheless, it serves me well by granting my two Crucial P3 the status of SSD NVMe instead of only Cache NVMe...

Before using your script:
![image](https://github.com/007revad/Synology_HDD_db/assets/29298312/8c16d9c8-2341-44f5-8ba2-b5495540e2fa)

After using your script:
![image](https://github.com/007revad/Synology_HDD_db/assets/29298312/b865a9fb-5029-4f77-8ca7-5afa7c9c6646)

Anyway, great work here! Thank you very much.

PS: one question though, do you think I must use the `--ram` parameter?

**Comment by 007revad** [comment]
Thank you. 

There is an online assemble link showing in both screenshots (in the Suggestion section). 

You only need to --ram option if you've installed more ram than Synology says is supported.

As I have Synology and Asustor NAS I've forked your Partage-UPS-Synology-avec-NAS-Asustor repo so I can add an English version of the readme.
